2025-04-23 16:22:41,245 WARNING: Prepared signature not found anywhere! [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:92]
2025-04-23 16:22:41,253 INFO: Tech Lead approval: {'stage': 1, 'approver_email': 'revanthmeda7702@gmail.com', 'title': 'Technical Lead', 'status': 'approved', 'timestamp': '2025-04-23T16:22:21.498947', 'signature': '0f14901d-e12c-41a8-9b9d-72414c82d2df_1.png', 'comment': '', 'approver_name': 'meda'} [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:120]
2025-04-23 16:22:41,254 INFO: Tech Lead signature path: C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\static\signatures\0f14901d-e12c-41a8-9b9d-72414c82d2df_1.png, exists: True [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:124]
2025-04-23 16:22:41,255 INFO: Created Tech Lead signature image [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:128]
2025-04-23 16:22:41,255 INFO: PM approval: {'stage': 2, 'approver_email': 'siriguppanihitha1998@gmail.com', 'title': 'Project Manager', 'status': 'approved', 'timestamp': '2025-04-23T16:22:41.220224', 'signature': '0f14901d-e12c-41a8-9b9d-72414c82d2df_2.png', 'comment': '', 'approver_name': 'poo'} [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:135]
2025-04-23 16:22:41,256 INFO: PM signature path: C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\static\signatures\0f14901d-e12c-41a8-9b9d-72414c82d2df_2.png, exists: True [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:139]
2025-04-23 16:22:41,256 INFO: Created PM signature image [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:143]
2025-04-23 16:22:41,256 ERROR: Error in approve_submission: name 'tech_lead_date' is not defined [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:242]
Traceback (most recent call last):
  File "C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py", line 164, in approve_submission
    "TECH_LEAD_DATE": tech_lead_date,
                      ^^^^^^^^^^^^^^
NameError: name 'tech_lead_date' is not defined. Did you mean: 'tech_lead_sig'?
2025-04-23 16:25:49,011 INFO: SAT Report Generator startup [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\app.py:38]
2025-04-23 16:26:58,281 INFO: Stored preparer signature as 3325e185-5d72-49e9-b26c-920f7e171712_prepared.png [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:131]
2025-04-23 16:26:58,281 INFO: Signature path: C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\static\signatures\3325e185-5d72-49e9-b26c-920f7e171712_prepared.png [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:132]
2025-04-23 16:26:58,282 INFO: File exists: True [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:133]
2025-04-23 16:27:09,602 WARNING: Could not autofit tables: 'in <string>' requires string as left operand, not ellipsis [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:538]
2025-04-23 16:27:11,900 INFO: Approval email sent to meda.revanth@gmail.com [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:558]
2025-04-23 16:27:11,925 INFO: Generated report for submission 3325e185-5d72-49e9-b26c-920f7e171712 [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:569]
2025-04-23 16:28:11,929 INFO: Notifying next approver: revanthmeda7702@gmail.com [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:215]
2025-04-23 16:28:29,098 INFO: Context keys: dict_keys(['DOCUMENT_TITLE', 'PROJECT_REFERENCE', 'DOCUMENT_REFERENCE', 'DATE', 'CLIENT_NAME', 'REVISION', 'REVISION_DETAILS', 'REVISION_DATE', 'PREPARED_BY', 'SIG_PREPARED', 'SIG_PREPARED_BY', 'REVIEWED_BY_TECH_LEAD', 'SIG_REVIEW_TECH', 'REVIEWED_BY_PM', 'SIG_REVIEW_PM', 'APPROVED_BY_CLIENT', 'SIG_APPROVAL_CLIENT', 'PURPOSE', 'SCOPE', 'PRE_TEST_REQUIREMENTS', 'KEY_COMPONENTS', 'IP_RECORDS', 'RELATED_DOCUMENTS', 'PRE_APPROVALS', 'POST_APPROVALS', 'SIGNAL_LISTS', 'ANALOGUE_LISTS', 'MODBUS_DIGITAL_LISTS', 'MODBUS_ANALOGUE_LISTS', 'DATA_VALIDATION', 'PROCESS_TEST', 'SCADA_VERIFICATION', 'TRENDS_TESTING', 'ALARM_LIST', 'SIG_APPROVER_1', 'SIG_APPROVER_2', 'SIG_APPROVER_3', 'approver_1_email', 'approver_2_email', 'approver_3_email']) [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:78]
2025-04-23 16:28:29,099 INFO: Submission keys: dict_keys(['context', 'user_email', 'approvals', 'locked', 'scada_image_urls', 'trends_image_urls', 'alarm_image_urls', 'created_at', 'updated_at', 'edit_notification_sent', 'approval_notification_sent']) [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:79]
2025-04-23 16:28:29,099 WARNING: Prepared signature not found anywhere! [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:92]
2025-04-23 16:28:29,100 INFO: Tech Lead approval: {'stage': 1, 'approver_email': 'meda.revanth@gmail.com', 'title': 'Technical Lead', 'status': 'approved', 'timestamp': '2025-04-23T16:28:11.900137', 'signature': '3325e185-5d72-49e9-b26c-920f7e171712_1.png', 'comment': '', 'approver_name': 'revanth'} [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:120]
2025-04-23 16:28:29,100 INFO: Tech Lead signature path: C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\static\signatures\3325e185-5d72-49e9-b26c-920f7e171712_1.png, exists: True [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:124]
2025-04-23 16:28:29,100 INFO: Created Tech Lead signature image [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:128]
2025-04-23 16:28:29,101 INFO: PM approval: {'stage': 2, 'approver_email': 'revanthmeda7702@gmail.com', 'title': 'Project Manager', 'status': 'approved', 'timestamp': '2025-04-23T16:28:29.072783', 'signature': '3325e185-5d72-49e9-b26c-920f7e171712_2.png', 'comment': '', 'approver_name': 'meda'} [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:135]
2025-04-23 16:28:29,101 INFO: PM signature path: C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\static\signatures\3325e185-5d72-49e9-b26c-920f7e171712_2.png, exists: True [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:139]
2025-04-23 16:28:29,101 INFO: Created PM signature image [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:143]
2025-04-23 16:28:29,101 ERROR: Error in approve_submission: name 'tech_lead_date' is not defined [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:242]
Traceback (most recent call last):
  File "C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py", line 164, in approve_submission
    "TECH_LEAD_DATE": tech_lead_date,
                      ^^^^^^^^^^^^^^
NameError: name 'tech_lead_date' is not defined. Did you mean: 'tech_lead_sig'?
2025-04-23 16:32:56,886 INFO: SAT Report Generator startup [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\app.py:38]
2025-04-23 16:33:34,131 INFO: Stored preparer signature as 6dfcc1ba-1586-4509-8e1e-41ad9237d14e_prepared.png [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:131]
2025-04-23 16:33:34,132 INFO: Signature path: C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\static\signatures\6dfcc1ba-1586-4509-8e1e-41ad9237d14e_prepared.png [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:132]
2025-04-23 16:33:34,133 INFO: File exists: True [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:133]
2025-04-23 16:33:44,906 WARNING: Could not autofit tables: 'in <string>' requires string as left operand, not ellipsis [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:538]
2025-04-23 16:33:47,044 INFO: Approval email sent to meda.revanth@gmail.com [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:558]
2025-04-23 16:33:47,073 INFO: Generated report for submission 6dfcc1ba-1586-4509-8e1e-41ad9237d14e [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\main.py:569]
2025-04-23 16:34:01,482 INFO: Notifying next approver: revanthmeda7702@gmail.com [in C:\Users\rmeda\OneDrive - CULLY AUTOMATION\Personal\Automate SAT\SAT_Report_App\routes\approval.py:247]
